### ISSUE TITLE
add streak info as a reward

### ISSUE THREAD

**Issue by specu** [sealemar/BingRewards#358]
it would be great for the script reported the streak information as a reward, including and especially whether the day's activities required to keep the streak going have been completed.

**Comment by S3phi40T** [comment]
@specu you know usually quizes are required for streak to move forward?

**Comment by specu** [comment]
yes, on tuesdays, wednesdays and thursdays (or mon, tue and wed). So that streak reward would show in the log whether manual intervention is required if the person running the script cared, and until quizzes are supported (#351)

**Comment by specu** [comment]
so basically streak is a 15 point per day bonus. the longer the streak the bigger than chance that if the streak breaks you get nothing.
